### DISCUSSION TITLE
How to control touch - displacement in mobile devices.

### DISCUSSION THREAD

**Discussion by javierfs** [sachinchoolur/lightGallery#1134] [Q&A]
I have been testing lightGallery on desktop devices and it works as I wish. However, the user experience for mobile devices is pretty poor. I am using an iPhone 12 and after clicking on the image, when light gallery is triggered, when I touch with two finger for zooming in, the image go crazy and zoom in x1000. Also, If I use the zoom icons, the resolution does not adapt to different zoom ins and remains blurry. This is a post in my website where you can see examples: https://www.visualdatalytics.com/2021-08-04-analyzing-playlists-dimensionality-reduction-unsupervised-machine-learning/

Also, it take a little while to load the images. I guess because they are a bit heavy. 

Do you have any idea on how can I fix these issues? 

Thanks a lot!

**Comment by sachinchoolur** [comment]
Hey @javierfs,

Looks like you are using lightGallery 1.x and it is no longer maintained. Could you please upgrade to version 2.x and let me know if you face any issues

**Reply by javierfs** [reply]
I will try to update it! Do u know any Hugo webstite that I can follow the folder structure and how to include the correct files? Thanks a lot!

**Reply by sachinchoolur** [reply]
To include lightGallery? You can try including directly from the `node_modules`. Or include files in the assets folder.
Let me know what issues are you facing with Hugo.
